generate-protos.sh

```diff
@@ -0,0 +1,13 @@
+set -ex
```

Comment:
Invoking this script locally produces a diff, like this:

```
@@ -1674,14 +1674,14 @@ _GETMETRICHISTORY = _descriptor.Descriptor(
       has_default_value=False, default_value=_b("").decode('utf-8'),
       message_type=None, enum_type=None, containing_type=None,
       is_extension=False, extension_scope=None,
-      options=_descriptor._ParseOptions(descriptor_pb2.FieldOptions(), _b('\210\265\030\001'))),
+      options=_descriptor._ParseOptions(descriptor_pb2.FieldOptions(), _b('\210\265\030\001')), file=DESCRIPTOR),
     _descriptor.FieldDescriptor(
       name='metric_key', full_name='mlflow.GetMetricHistory.metric_key', index=1,
       number=2, type=9, cpp_type=9, label=1,
       has_default_value=False, default_value=_b("").decode('utf-8'),
       message_type=None, enum_type=None, containing_type=None,
       is_extension=False, extension_scope=None,
-      options=_descriptor._ParseOptions(descriptor_pb2.FieldOptions(), _b('\210\265\030\001'))),
+      options=_descriptor._ParseOptions(descriptor_pb2.FieldOptions(), _b('\210\265\030\001')), file=DESCRIPTOR),
   ],
   extensions=[
   ],
@@ -2284,6 +2284,9 @@ _MLFLOWSERVICE = _descriptor.ServiceDescriptor(
     options=_descriptor._ParseOptions(descriptor_pb2.MethodOptions(), _b('\202\265\0304\n0\n\003GET\022#/preview/mlflow/metrics/get-history\032\004\010\002\020\000\020\001')),
   ),
 ])
+_sym_db.RegisterServiceDescriptor(_MLFLOWSERVICE)
+
+DESCRIPTOR.services_by_name['MlflowService'] = _MLFLOWSERVICE

 MlflowService = service_reflection.GeneratedServiceType('MlflowService', (_service.Service,), dict(
   DESCRIPTOR = _MLFLOWSERVICE,
```

But it does that Descriptor change for every message. How can we make sure this process is deterministic so that anyone can run the command?

Reply:
btw:

```
$ protoc --version
libprotoc 3.5.1
```

Reply:
Ah, I have
```
$ protoc --version
libprotoc 3.3.0
```